library/common-library/templates/_low-data-mode.tpl

```diff
@@ -0,0 +1,26 @@
+{{- /*
+Abstraction of the lowDataMode toggle.
+This helper allows to override the global `.global.lowDataMode` with the value of `.lowDataMode`.
+Returns "true" if `lowDataMode` is enabled, otherwise "" (empty string)
+*/ -}}
+{{- define "common.lowDataMode" -}}
+{{- /* `get` will return "" (empty string) if value is not found, and the value otherwise, so we can type-assert with kindIs */ -}}
+{{- if (get .Values "lowDataMode" | kindIs "bool") -}}
```

Comment:
This logic is also used for [creating the service account](https://github.com/newrelic/helm-charts/pull/713/files#diff-4b3d46fb917e8d56b8de51a27e90a24a9a0eff7215b74052d67360f8483730e2R1-R28) and [it is tested](https://github.com/newrelic/helm-charts/pull/713/files#diff-37a891818928aebca3dd07fc22b73a546722dc6c509067e4f9cfb7cfb9f6a028)

But you are completely right, that test should exist too. What I don't know is how because it should template something like an example of configMap. Do you have any idea of an example you would like to have?

Reply:
I would add the most independent and short template to test this logic , like an unit test. But what i mean was more general , let me rephrase it , are we going to have unit test on the library chart?